app/forms/hyrax/forms/resource_form.rb

```diff
@@ -15,9 +15,7 @@ def self.ResourceForm(work_class)
       Class.new(Hyrax::Forms::ResourceForm) do
         self.model_class = work_class
 
-        (work_class.fields - work_class.reserved_attributes).each do |field|
-          property field, default: nil
-        end
+        include Hyrax::FormFields(:core_metadata)
```

Comment:
This line is so much nicer! The indirection helps keep abstraction at the same leve